spec/controllers/api/scores_controller_spec.rb

```diff
@@ -104,4 +110,12 @@
       expect(Score.count).to eq score_count
     end
   end
+
+  it 'should return at most 25 scores' do
+    get user_feed_path
```

Comment:
We should use the rails path helpers: https://guides.rubyonrails.org/routing.html#generating-paths-and-urls-from-code